phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/assoc/FileUtils.java

```diff
@@ -0,0 +1,17 @@
+package org.monarchinitiative.phenol.annotations.assoc;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.zip.GZIPInputStream;
+
+class FileUtils {
+
+  static BufferedReader openForReading(Path geneInfoFile) throws IOException {
```

Comment:
Why not call this `FileUtils.newBufferedReader` ? Needs a brief Javadoc description as to what this method does / how this differs from `Files`

Reply:
Thanks!